scripts/deploy-staking-only.js

```
@@ -0,0 +1,78 @@
+const {ethers} = require("hardhat");
+
+async function main() {
+    const INDEX = 269238508004; // Exact mainnet index
+    const EPOCH_LENGTH = 28800; // 8 hours
+    const OHM = "0x784cA0C006b8651BAB183829A99fA46BeCe50dBc";
+    const TREASURY = "0x7D7406e4E5Fdb636C888cF17aBb42B5edE8B3722";
+    const AUTHORITY = "0x81057Bef097462957B9388D8DCB7D4AB0699cADB";
+
+    const [deployer] = await ethers.getSigners();
+    console.log("Deploying with:", deployer.address);
+    console.log("Network:", network.name);
```

ESLint

[error] 'network' is not defined.

(no-undef)